frontend/public/components/events.jsx

```diff
@@ -52,18 +59,19 @@ class Inner extends React.PureComponent {
               name={obj.name}
               title={obj.uid}
             />
+            <small className="co-sysevent__source">
+              Generated from <span>{source.component}</span>
+              {source.component === 'kubelet' &&
```

Comment:
Explain `source.component === 'kubelet'`.  `Inner` seems to be a pretty generic component, I don't know that it should know anything specific about any particular other kind of component.

Reply:
Update: nevermind, I see this existed previously, you just moved it around.  Leaving this comment history so I don't forget and ask again later :)

Reply:
> Inner seems to be a pretty generic component, I don't know that it should know anything specific about any particular other kind of component.

This is checking if the source of the k8s event is the kubelet. It's a property on the event itself.

Reply:
cool, thx!